doc/conf.py

```diff
@@ -32,31 +33,43 @@
 
 sys.path.append(abspath(dirname(__file__)))
 import mocks
+mocks.import_fake_modules()
+mocks.fake_xl_settings()
 sys.path.pop()
 
 
-# -- General configuration ----------------------------------------------------
+# -- General configuration ------------------------------------------------
 
-# Add any Sphinx extension module names here, as strings. They can be extensions
-# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autosummary', 'sphinx.ext.autodoc',
-              'sphinx.ext.doctest', 'sphinx.ext.todo', 'sphinx.ext.coverage']
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.todo',
+    'sphinx.ext.coverage',
+    'sphinx.ext.viewcode']
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
-# The suffix of source filenames.
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
 source_suffix = '.rst'
 
-# The encoding of source files.
-source_encoding = 'utf-8'
-
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
 project = u'Exaile'
-copyright = u'2009-2014 Exaile Development Team'
+copyright = u'2009-2017 Exaile Development Team'
+author = u'Adam Olsen <arolsen@gmail.com>, Johannes Sasongko <sasongko@gmail.com>, Aren Olsen <reacocard@gmail.com>, Mathias Brodala <info@noctus.net>, Christian Stadelmann'
```

Comment:
Oops, I expected the document to be so old, that you were included already… (copy-pasted from below, where the same mistake is, a second time)